src/Document.js

```diff
@@ -73,6 +76,18 @@ export default class Document extends Node {
             case 'option':
                 element = new HTMLOptionElement();
                 break;
+            case 'table':
+                element = new HTMLTableElement();
+                break;
+            case 'thead':
+            case 'tfoot':
+            case 'tbody':
+                element = new HTMLTableSectionElement();
```

Comment:
Because they have common interface as written in documentation. See https://developer.mozilla.org/en-US/docs/Web/API/HTMLTableSectionElement